foundations/html_css/html-foundations/links-and-images.md

```diff
@@ -50,6 +50,24 @@ By default, any text wrapped with an anchor tag without a `href` attribute will
 
 It's worth noting you can use anchor tags to link to any kind of resource on the internet, not just other HTML documents. You can link to videos, pdf files, images, and so on, but for the most part, you will be linking to other HTML documents.
 
+### Opening links in a new tab
+
+The method shown above opens links in the same tab as the webpage containg them. This is the default behaviour of most browsers and it can be changed relatively easily. All we need is another attribute: the `target` attribute. 
```

Comment:
Small typo. It should be `containing`

Reply:
Hi. I have pushed the changes to my local branch and the changes do seem to appear on [my forked github repo](https://github.com/Shadow5634/curriculum/tree/links_in_new_tab). Github says that this branch is **not behind the upstream TheOdinProject:main** and that it is **[5 commits ahead](https://github.com/Shadow5634/curriculum/compare/TheOdinProject:curriculum:main...links_in_new_tab) of TheOdinProject:main**

However I do not see the commit on this pull request. Is there anything that I am missing? I remember that when i ran git fetch upstream some change to a clean_code.md file came in but not sure how that is relevant.